script.js

```diff
@@ -0,0 +1,81 @@
+let rootDiv = document.getElementById("root");
+let gridsize = 16; //default
+let clr = document.getElementById("clr");
+let radioB1 = document.getElementById("radioBut1");
+let radioB2 = document.getElementById("radioBut2");
+let radioB3 = document.getElementById("radioBut3");
+let randClr = "",
+  erase = false;
+let clrS = document.getElementById("clrSelected");
+
+function drawGrid(grdSize) {
+  rootDiv.innerHTML = "";
+  for (let i = 0; i < grdSize * grdSize; i++) {
+    let divEl = document.createElement("div");
+    divEl.setAttribute("class", "divElement");
+    divEl.style.width = 960 / grdSize + "px";
+    divEl.style.height = 960 / grdSize / 1.8 + "px";
+
+    divEl.addEventListener("mouseover", function (e) {
+      if (erase == true) {
+        e.target.style.backgroundColor = "";
+      } else {
+        if (radioB1.checked) {
+          //random colors
+          divEl.style.backgroundColor =
+            "#" + Math.random().toString(16).substr(-6);
+        } else if (radioB2.checked) {
+          //random color
+          divEl.style.backgroundColor = randClr;
+        } else if (radioB3.checked) {
+          //picked color
```

Comment:
leaving a comment here is great :+1: for each functionality add a comment there too, so I can get a better clue for what's your strategy of solving the challenge

Reply:
ok